src/editors/Flow/view/Flow.jsx

```diff
@@ -1380,13 +1384,6 @@ export const Flow = (props, ref) => {
         if (viewMode === FLOW_VIEW_MODE.treeView && docData.doc.name === name) {
           const subFlows = mainInterfaceRef.current.graph.subFlows;
 
-          if (!docData.thisDoc.isDirty) {
```

Comment:
This is removed on purpose? Why?

Reply:
The if was not making sense. If no changes identified then safe. Some dead code right there. If you can justify me why the code was there or its intention i would add it back.

Reply:
Oh I see what you mean. Thanks